src/telemetry/client.hpp

```diff
@@ -23,7 +23,7 @@
 
 #include <google/protobuf/io/zero_copy_stream.h>
 #include <string>
-#include "types/message.pb.h"
+#include "telemetrydata/message.pb.h"
```

Comment:
make all include paths starting from the root `src` folder, so this one should be something line `#include "telemetry/telemetrydata/message.pb.h"`

Reply:
Fixed